x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_utilities.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { EuiBreadcrumbs, Breadcrumb } from '@elastic/eui';
+import styled from 'styled-components';
+import React, { memo } from 'react';
+import { useResolverTheme } from '../assets';
+
+/**
+ * The two query parameters we read/write on to control which view the table presents:
+ */
+export interface CrumbInfo {
+  readonly crumbId: string;
+  readonly crumbEvent: string;
+}
+
+const ThemedBreadcrumbs = styled(EuiBreadcrumbs)<{ background: string; text: string }>`
+  &.euiBreadcrumbs.euiBreadcrumbs--responsive {
+    background-color: ${(props) => props.background};
+    color: ${(props) => props.text};
+    padding: 1em;
+  }
+`;
+
+/**
+ * Breadcrumb menu with adjustments per direction from UX team
+ */
+export const StyledBreadcrumbs = memo(function StyledBreadcrumbs({
+  breadcrumbs,
+  truncate,
+}: {
+  breadcrumbs: Breadcrumb[];
+  truncate?: boolean;
+}) {
+  const {
+    colorMap: { resolverEdge, resolverEdgeText },
+  } = useResolverTheme();

```

Comment:
@michaelolo24 am I doing this right?

Reply:
Yep! Feel free to change any property names or things if it seems confusing in any way